packages/abap-deploy-config-inquirer/src/prompts/conditions.ts

```diff
@@ -182,12 +182,15 @@ function defaultOrShowPackageQuestion(): boolean {
 /**
  * Determines if the choice of package input options should include both manual input and search (autocomplete) input options.
  *
+ * @param useAutocomplete - useAutocomplete option from prompt options
  * @returns boolean
  */
-export function showPackageInputChoiceQuestion(): boolean {
-    // Only show the input choice (manual/search) when the autocomplete prompt is supported; CLI or YUI specific version
+export function showPackageInputChoiceQuestion(useAutocomplete = false): boolean {
+    // Only show the input choice (manual/search) when the autocomplete option is true, the prompt is supported; CLI or YUI specific version
     return Boolean(
-        (!PromptState.isYUI || isFeatureEnabled('enableAutocompleteUIPrompt')) && defaultOrShowPackageQuestion()
+        (!PromptState.isYUI || isFeatureEnabled('enableAutocompleteUIPrompt')) &&
```

Comment:
Not clear why we are using feature toggle for an option?

Reply:
Removed as discussed